pkg/database/database.go

```diff
@@ -34,16 +34,18 @@ type DBInterface interface {
 }
 
 type Database struct {
-	db                *sql.DB
-	resourceTableName string
+	db   *sql.DB
+	info DatabaseInfo
```

Comment:
This change will require changes to the test code (pkg/database/database_test.go), which I do not see in this MR.

Reply:
This is a draft, and essentially @maruiz93 created it for me to look at.

Reply:
Sorry - I missed the Draft label.